app/controllers/comments.controllers.js

```diff
@@ -28,3 +28,12 @@ exports.postCommentById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const comment_id = req.params.comment_id
+  const commentExistsQuery = checkCommentExists(comment_id);
```

Comment:
No need to check if the comment exists first, lets look at the response that comes back from the delete query to determine if it has successfully deleted something 👍